docs/egress.md

```diff
@@ -127,6 +131,17 @@ The `externalIPPool` field specifies the name of the `ExternalIPPool` that the
 be assigned to. It can be empty, which means users should assign the `egressIP`
 to one Node manually.
 
+### RateLimit
+
+The `rateLimit` field impose rate-limiting on egress traffic of the Egress. `peak`
+specifies the maximum mbps. `burst` specifies maximum burst mbps for throttle.
+All backend workloads selected by a rate-limited Egress share the same bandwidth
+while sending egress traffic via this Egress.
```

Comment:
Not sure if it's just me, I thought the sentence means every workload has its own exclusive bandwidth and the value is same until I see the code.